velox/functions/prestosql/window/tests/WindowTestBase.cpp

```diff
@@ -109,35 +92,34 @@ VectorPtr WindowTestBase::makeFlatFuzzVector(
   return fuzzer.fuzzFlat(type);
 }
 
-std::vector<std::string> WindowTestBase::addSuffixToClauses(
-    const std::string& suffix,
-    const std::vector<std::string>& inputClauses) {
-  std::vector<std::string> output;
-  output.reserve(inputClauses.size());
-  for (auto input : inputClauses) {
-    output.push_back(input + suffix);
-  }
-  return output;
+RowVectorPtr WindowTestBase::makeRandomInputVector(vector_size_t size) {
+  return makeRowVector(
+      {makeFlatFuzzVector(BIGINT(), size, 0.2),
+       makeFlatFuzzVector(VARCHAR(), size, 0.3),
+       makeFlatVector<int64_t>(size, [](auto row) { return row % 5 + 1; }),
```

Comment:
Actually, It would be good to make this random. But I couldn't come up with a clean way to generate a random int64 vector with values > 0. Any ideas ?

Reply:
Thanks @mbasmanova for suggestion to use boost random_int_distribution. Have updated the code.